what_do_we_offer_styles.css

```diff
@@ -1,4 +1,7 @@
 @import url(fonts.css);
+body {
+    background-color: #FFFFFF;
+}
```

Comment:
@IharDorinel I think it's not important to set white color because it always sets by default